fastapi_template/cli.py

```diff
@@ -312,6 +312,12 @@ def checker(ctx: BuilderContext) -> bool:
     multiselect=True,
     before_ask=do_not_ask_features_if_quite,
     entries=[
+        MenuEntry(
+            code="pydanticv1",
+            cli_name="pydantic-v1",
+            user_view="Disable pydantic settings",
```

Comment:
It will change not only settings, but all models. 

You can set it to "Use older version of pydantic" for example.